### PR TITLE
v1.0 merge

### PR DESCRIPTION
**Pull request by teckim**
### In this pull request:

- Created models according to the ERD diagram.
- Implement the authentication and authorization.
- Created the page to add a category.
- Created the home page.
- Created the page to add a transaction.
- Created the transactions page.
- Made sure the app is tested adequately.
- Deployed the project.
- Recorded a video for the project.
- Created a good README and PR description.

> [Video presentation](https://www.loom.com/share/7454254085c84b19b88a3b696a0fdfb9)

> _Note: Please run `rails db:seed RAILS_ENV=test` before running the tests_

### CODE REVIEW

File: config/environments/development.rb [hunk 1]
```diff
@@ -67,4 +67,15 @@
...
+  config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.smtp_settings = {
+    :user_name => 'a5c8d8c55c885a',
+    :password => '975f061dea3481',
+    :address => 'smtp.mailtrap.io',
+    :domain => 'smtp.mailtrap.io',
+    :port => '2525',
+    :authentication => :cram_md5
+  }
```
Comment (omar-muhamad) [inline config/environments/development.rb:80]: - [ ] When I try to sign up a new user it says the confirmation message has been sent to the email but I didn't get any confirmation emails. So please fix that.

image

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @teckim,

Nice job, you are almost there.



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used the correct workflow.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please deploy the application using Heroku, Netlify, or GitHub Pages. (recommended: Heroku), and kindly add the deployment link to the README file.

- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#:~:text=Once%20you%20have,very%20proud%20of.), Please add the video presentation in the code review summary.

- [ ] There are 4 tests not passing, so please fix that.

image

- [ ] Please check the inline comments below.

## [Optional] suggestions:

Please add these instructions to the `get started` section in your readme file:
-  Run `rails assets:precompile` to make the tailwind asset file works.
- Run `rails db:migrate RAILS_ENV=test` before `rails db:seed RAILS_ENV=test` to make your tests work.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: spec/system/auth_spec.rb [hunk 2]
```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
```
Comment (GzimAsani) [inline spec/system/auth_spec.rb]: - [ ] There are 6 failed tests, kindly update them 
image

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello    @teckim      :wave:   
 Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * You deployed your app online  :+1:
 * You added the presentation to the PR summary  :+1:
 * I can confirm the email when singing  :+1:

 ## Required Changes 

- [ ] Check the comment below :arrow_double_down: 






 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: spec/system/auth_spec.rb (hunk 2 above)
Comment (teckim) [reply spec/system/auth_spec.rb]: > * [ ]  There are 6 failed tests, kindly update them
>   image

Hello @GzimAsani, have you seeded the test database? if not please follow the test instructions in the readme file.

Comment (GzimAsani) [reply spec/system/auth_spec.rb]: Hi @teckim, yes I did

Comment (teckim) [reply spec/system/auth_spec.rb]: > Hi @teckim, yes I did

That seems incorrect! it works fine in my end?

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi @teckim ,
# STATUS: APPROVED! 🥇 💯 
done

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**